website/views/options/social/chat-box.jade

```diff
@@ -14,7 +14,7 @@ form.chat-form(ng-if='user.flags.communityGuidelinesAccepted' ng-submit='postCha
     textarea.form-control(rows=4, ui-keydown='{"meta-enter":"postChat(group,message.content)"}', ui-keypress='{13:"postChat(group,message.content)"}', ng-model='message.content', updateinterval='250', flag='@', at-user, auto-complete placeholder="{{group._id == 'habitrpg' ? env.t('tavernCommunityGuidelinesPlaceholder') : ''}}", ng-disabled='_sending == true')
     span.user-list
       ul.list-at-user(ng-show="query")
-        li(ng-repeat='msg in response | filter:filterUser | limitTo: 5', ng-click='performCompletion(msg)')
+        li(ng-repeat='msg in response track by $index | filter:filterUser | limitTo: 5', ng-click='performCompletion(msg)')
```

Comment:
This is in the wrong spot and breaks the `limitTo`

![screen shot 2015-06-26 at 10 49 50 pm](https://cloud.githubusercontent.com/assets/2916945/8390374/ba2f5e00-1c55-11e5-8032-75398a1de3a9.png)